### ISSUE TITLE
Automate PDF Generation

### ISSUE THREAD

**Issue by samypr100** [trinity-xai/Trinity#19]
Auto generate `src/main/resources/edu/jhuapl/trinity/wiki/Trinity.pdf` during build to be able to remove it from version control

**Comment by czp13** [comment]
Generate it from the source of this md file, this way the pdf would be more dynamic as well, and less space we use on GitHub.:
- `src/main/resources/edu/jhuapl/trinity/wiki/Trinity.md`
Can this be a source of truth? I can pick this up potentially if we feel, seems straightforward.

**Comment by samypr100** [comment]
Indeed

**Comment by czp13** [comment]
> [@samypr100](https://github.com/samypr100) what was the original intent of this issue? If it was to simply remove the pdf file from the src/resources tree then lets just do that. we can make it available elsewhere.

I guess that was the intent as we version control the file, it is just waste of resources.

**Comment by Birdasaur** [comment]
this is being address in #13 starting with commit :
https://github.com/trinity-xai/Trinity/pull/118/commits/4321f713e69100141f10ec0fb2a4e60ac0e39f01
Will close this issue once that PR is merged into main.
